joomla/joomla-cms
# a11y improvement for login form

Every form with a fieldset should also have a legend. The login form does not
### Summary of Changes

Add a legend to the fieldset using a new language string and with a css class that renders the text invisible.
### Testing Instructions

Apply PR and go to login page (not module) on the front end
Check the code and you will find after the fieldset tag a legend tag

`<legend class="element-invisible">Login Form</legend>`
